lib/components/user/monitored-trip/trip-basics-pane.tsx

```diff
@@ -377,6 +382,7 @@ class TripBasicsPane extends Component<TripBasicsProps, State> {
           <FormGroup validationState={errorStates.tripName}>
             <ControlLabel htmlFor="tripName">
               <FormattedMessage id="components.TripBasicsPane.tripNamePrompt" />
+              {!isReadOnly && <RequiredIndicator>{' *'}</RequiredIndicator>}
             </ControlLabel>
             {/* onBlur, onChange, and value are passed automatically. */}
             <Field
```

Comment:
Should we add a `required` attribute to this field?

Reply:
Wow I'm a little surprised to realize we didn't have the `required` attribute on here already! Thank you for catching this, I agree it's a much better approach